drivers/wifi/esp_at/esp.c

```diff
@@ -825,7 +825,7 @@ static int cmd_ipd_parse_hdr(struct esp_data *dev,
 		char *remote_ip;
 		long port;
 
-		err = esp_pull_quoted(&str, str_end, &remote_ip);
+		err = esp_pull_raw(&str, str_end, &remote_ip);
```

Comment:
This is what I get after printing `LOG_HEXDUMP_INF(str, str_end - str, "remote_ip before parse");`:
```
[00:00:09.239,776] <inf> wifi_esp_at: remote_ip before parse
                                      22 31 30 2e 31 31 2e 31  32 2e 31 22 2c 35 33 3a |"10.11.1 2.1",53:
                                      72 80 81 80 00 01 00 01  00 00 00 00 04 63 6f 61 |r....... .....coa
                                      70
``` 
so IP is quoted.

Which ESP-AT firmware version do you use?

Reply:
AT version is 1.7.0:
```
[00:00:00.100,000] <inf> wifi_esp_at: Waiting for interface to come up
[00:00:00.422,000] <inf> wifi_esp_at: AT version: 1.7.0.0(Aug 16 2018 00:57:04)
[00:00:00.425,000] <inf> wifi_esp_at: SDK version: 3.0.0(d49923c)
[00:00:00.799,000] <inf> wifi_esp_at: ESP Wi-Fi ready
```

Reply:
I am using 2.1.0 or 2.2.0 I think. This means that we need to handle both cases, quoted and unquoted.

Reply:
Updated to handle AT version discrepancy